internal/clients/telegram/client.go

```diff
@@ -58,6 +59,7 @@ which are going to process all updates in parallel
 This function considers 0 or less goroutines as fatal.
 */
 func (c *Client) Start(goroutines uint) {
```

Comment:
do we really need this functionality with goroutines in an argument? why?

Reply:
I think this is best to be addressed in #2.

Reply:
Because of this comment I am assuming this is resolved

https://github.com/m-kuzmin/daily-reporter/pull/2#issuecomment-1586856669